frontend/javascripts/oxalis/shaders/main_data_fragment.glsl.ts

```diff
@@ -251,5 +284,249 @@ void main() {
       `vec3(${vector3.map(formatNumberAsGLSLFloat).join(", ")})`,
     OrthoViewIndices: _.mapValues(OrthoViewIndices, formatNumberAsGLSLFloat),
     hasSegmentation,
+    isFragment: true,
+  });
+}
+
+export function getMainVertexShader(params: Params) {
```

Comment:
The name of the file suggests that it only contains a fragment shader. It should be adapted.